Runtime/2015-09-08-runtime.md

```diff
@@ -21,7 +21,12 @@
 ### LAMB - Erik Jasiak
 
 ### UAA - Sree Tummidi
-
+- Work continues on supporting invitations for external user stores via LDAP & SAML. Invitations API will be exposed to allow inviting external users to the CF Platform.
+- Consolidating the allowed Redirect URIs for OAuth Clients to protect against open redirects. Earlier there were separate fields for Sign Up & Change Email and Invitations. Now all these will be consolidated and Log Out URIs will be supported on each oAuth client.
+- The team has worked on proposals for UAA & SAML Integration and support for mapping CF roles to SAML & LDAP Groups/Roles. These proposals will be shared with the foundation for their feedback & review. : 
+  - [UAA & SAML Integration](https://docs.google.com/document/d/107sv7YqxdoDWi2vX5Z8WHm1JaqwHZOL_wa-esn2U5cE/edit)
```

Comment:
@sreetummidi 

Thanks for sharing this update. The "UAA & SAML proposal" and "CF Roles & LDAP/SAML Integration" documents seem to be lacking read/comment permissions in google docs, any chance to fix that ?